app/scripts/modules/ecs/src/serverGroup/events/events.html

```diff
@@ -11,21 +11,19 @@ <h3>Server Group Events for {{$ctrl.serverGroup.name}}</h3>
       <p>There was an error loading events for {{$ctrl.serverGroup.name}}. Please try again later.</p>
     </div>
     <div ng-if="!$ctrl.viewState.loading && !$ctrl.viewState.error && !$ctrl.events.length" class="text-center">
-      No scaling events found for {{$ctrl.serverGroup.name}}.</p>
+      <p>No scaling events found for {{$ctrl.serverGroup.name}}.</p>
     </div>
 
     <div ng-repeat="event in $ctrl.events" ng-if="!$ctrl.viewState.loading">
-        <p class="clearfix">
-          <div ng-if="event.status ==git ad= 'Success'">
+      <p class="clearfix"></p>
+          <div ng-if="event.status === 'Success'">
```

Comment:
I dont know how these angular conditionals work but this change looks suspicious

Reply:
The old markup is nonsense and I suspect a typo:  
```
 <div ng-if="event.status ==git ad= 'Success'">
```

Reply:
right on